docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -55,6 +55,29 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <10> In order to release all transient resources (memory, threads, ...) a `CacheManager` provides to `Cache` instances
      it manages, you have to invoke `CacheManager.close()`, which in turns closes all `Cache` instances known at the time.
 
+=== Clustered cache manager
+
+In addition to the steps performed for creating standalone cache manager, clustered cache manager creation requires you to provide the clustering
```

Comment:
I'm not sure we want to refer to a "clustered cache manager", so much as a cache manager that supports clustered caches.  Something like: "To support the creation of clustered caches, firstly you must provide a clustering service configuration", but maybe not quite so abrupt